src/navigationGuards.ts

```diff
@@ -298,9 +298,13 @@ export function extractComponentsGuards(
           componentPromise = Promise.resolve(componentPromise as RouteComponent)
         } else {
           // display the error if any
-          componentPromise = componentPromise.catch(
-            __DEV__ ? err => err && warn(err) : console.error
-          )
+          componentPromise = componentPromise.catch(err => {
+            if (__DEV__ && err) {
+              warn(err)
+            }
+
+            console.error(err)
```

Comment:
let's just keep the `console.error` if the warn is useless in dev

Reply:
Sounds good to me

Reply:
Done.